src/google/protobuf/descriptor.cc

```diff
@@ -5974,8 +5980,7 @@ void DescriptorBuilder::CheckEnumValueUniqueness(
     const EnumValueDescriptor* value = result->value(i);
     std::string stripped =
         EnumValueToPascalCase(remover.MaybeRemove(value->name()));
-    std::pair<absl::flat_hash_map<std::string, const EnumValueDescriptor*>::iterator, bool>
-        insert_result = values.insert(std::make_pair(stripped, value));
+    auto insert_result = values.insert(std::make_pair(stripped, value));
```

Comment:
clang-format blew this type up considerably, so it seemed far easier to read to just use auto here,. Is that okay?

Reply:
For insert, I think `auto` is very reasonable.